homeassistant/components/watts/climate.py

```diff
@@ -0,0 +1,161 @@
+"""Climate platform for Watts Vision integration."""
+
+from __future__ import annotations
+
+import asyncio
+import logging
+from typing import Any
+
+from visionpluspython.models import ThermostatDevice
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from . import WattsVisionConfigEntry
+from .const import (
+    HVAC_MODE_TO_THERMOSTAT,
+    THERMOSTAT_MODE_TO_HVAC,
+    UPDATE_DELAY_AFTER_COMMAND,
+)
+from .coordinator import WattsVisionCoordinator
+from .entity import WattsVisionEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: WattsVisionConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up Watts Vision climate entities from a config entry."""
+
+    coordinator: WattsVisionCoordinator = entry.runtime_data.coordinator
```

Comment:
```suggestion
    coordinator = entry.runtime_data.coordinator
```
It can infer the type